proof/crefine/RISCV64/Machine_C.thy

```diff
@@ -130,6 +130,539 @@ lemma dmo_if:
   "(doMachineOp (if a then b else c)) = (if a then (doMachineOp b) else (doMachineOp c))"
   by (simp split: if_split)
 
-end
+(* Count leading and trailing zeros. *)
+
+(* FIXME: move *)
```

Comment:
Move all the FIXME moves.

Reply:
Do I need to? I'm not sure if other parts of l4v would make use of these.

Reply:
Done.